src/client/app/containers/import.ts

```diff
@@ -213,4 +219,36 @@ export class Import {
     cancel() {
         this._store.dispatch(new UI.ToggleImportAction(false));
     }
+
+    checkForViewSnippet() {
+        if (!this.hasViewSettings()) {
+            return;
+        }
+
+        let viewData = Office.context.document.settings.get('VIEW_URL_SETTINGS');
```

Comment:
Also now that we support Outlook, I believe Office.context.document is not available for Outlook add-ins.